curator-client/src/main/java/org/apache/curator/utils/Compatibility.java

```diff
@@ -88,7 +88,7 @@ public static boolean hasAddrField()
         return (addrField != null);
     }
 
-    public static String getHostAddress(QuorumPeer.QuorumServer server)
+    public static String getHostString(QuorumPeer.QuorumServer server)
```

Comment:
I think it is ok for this renaming as `Compatibility` is not intended for external usage .